codebase2/codebase-sqlite/U/Codebase/Sqlite/Queries.hs

```diff
@@ -414,27 +419,11 @@ type TextPathSegments = [Text]
 -- * main squeeze
 
 currentSchemaVersion :: SchemaVersion
-currentSchemaVersion = 16
+currentSchemaVersion = 17
 
-createSchema :: Transaction ()
-createSchema = do
```

Comment:
`createSchema` was moved to `Unison.Codebase.SqliteCodebase.Operations` to prevent cyclic imports.